apps/web/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "web",
-	"version": "1.1.0",
+	"version": "1.1.1",
```

Comment:
fyi, I'm gonna start doing semantic versioning for this project just to avoid headaches in the future.

![image](https://github.com/stamford-syntax-club/course-compose/assets/92321280/bc71c607-3e01-4b0f-b20b-7ecd90eeeb05)
![image](https://github.com/stamford-syntax-club/course-compose/assets/92321280/21165999-2809-4540-9c5e-8aed69736b42)